tests/system/action/organization/test_initial_import.py

```diff
@@ -0,0 +1,531 @@
+from typing import Any, Dict
+
+from tests.system.action.base import BaseActionTestCase
+
+
+class OrganizationInitialImport(BaseActionTestCase):
+    def create_request_data(self, datapart: Dict[str, Any]) -> Dict[str, Any]:
+        data: Dict[str, Any] = {
```

Comment:
I think it would be better to fetch the initial data here (once, for all tests), analogous to the example data we use elsewhere from a fixed commit. This would make the tests way less verbose.